deep_qa/layers/attention/attention.py

```diff
@@ -32,16 +33,19 @@ class Attention(MaskedLayer):
 
     Parameters
     ----------
-    similarity_function_params: "Params", optional (default={})
+    similarity_function_params: Dict, optional (default={})
         These parameters get passed to a similarity function (see
         :mod:`deep_qa.tensors.similarity_functions` for more info on what's acceptable).  The
         default similarity function with no parameters is a simple dot product.
     """
-    def __init__(self, similarity_function: "Params"=Params({}), **kwargs):
+    # pylint: disable=dangerous-default-value
+    def __init__(self, similarity_function: Dict={}, **kwargs):
```

Comment:
Yeah, this should just go back to how it was, without the pylint warning, and with a default value of `None`.